changes/ 19332-clear-secrets-with-gitops

```diff
@@ -0,0 +1 @@
+Enable gitops to create teams with no enroll secrets, or clear enroll secrets for an existing team by setting team_settings.secret to nothing or to an empty array.
```

Comment:
Nit: Missing bullet `*` (for automation of changelog)

Also, do we need a list item for the change in `fleetctl apply`?